src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -79,3 +79,118 @@ def cli(self, output=None):
 
         return ret_dict
 
+
+
+# ===================
+# Schema for:
+#  * 'show snmp user'
+# ===================
+class ShowSnmpUserSchema(MetaParser):
+    """Schema for show snmp user."""
+
+    schema = {
+        Any(): {
+            Optional("access-list"): str,
```

Comment:
we usually use `_` in key name for iosxe. please change to from `-` to `_`.

Reply:
Replaced hyphens with underscores.

Reply:
confirmed